### PR TITLE
enh: swap libpysal in for pysal

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

**Comment by ljwolf** [comment]
Ah, thanks, I thought I'd removed all this, but I see now I haven't. There're a few spots elsewhere I'll have to swap. I'll add more commits to this this week.

**Comment by ljwolf** [comment]
[a few swaps](https://github.com/pysal/spvcm/search?utf8=%E2%9C%93&q=pysal&type=)
- [ ] `testing`
- [ ] `utils`
- [ ] `both_levels`

**Comment by ljwolf** [comment]
doesn't matter, never had it :smile:

**Comment by ljwolf** [comment]
I've rebased this and cleaned up a few remaining references. I will keep nursing this through the CI (I see a few numerical test failures locally, could just be the peculiar numpy build I'm using).

Once done, I'll release as `0.2post2` on github & pypi, and we'll use that to build the pysal release.

**Comment by ljwolf** [comment]
Large footprint in terms of size? I think this is because of the csvs I'm shipping around for testing. I will see how I can reduce that size, probably by turning those into `csv.gz` would help. '

If you mean in terms of test time, one of the tests (estimating the spectral variance of the markov chain) takes a very long time using `statsmodels`, but has code to use R's `coda` package if `rpy2` and `coda` are available. My travis setup includes all that's necessary to make the fast versions run, but I understand if that's not something we want to do in the full package.

**Comment by sjsrey** [comment]
size is more of the issue.
Sergio (Serge) Rey
Founding Director, Center for Geospatial Sciences
Professor, School of Public Policy
University of California, Riverside
sergio.rey@ucr.edu

**Comment by ljwolf** [comment]
OK, will work on this angle.

**Comment by ljwolf** [comment]
OH wow I didn't catch that the last tests errorred out because they took to long to install `r-essentials` and `r-coda`. This is... not ideal.

**Comment by ljwolf** [comment]
This is fixed. Gzipping the archives changed the testing results... I'm not sure why. So, I'd like to keep them unarchived for now. 2 possible solutions:
1. Can we force them to be ignored for the `MANIFEST.in?` 
2. Some of the stuff that takes a while to *clone* is the git history, and you could force that to be shallow using the `--depth` argument. 
3. is this fixed because we're hitting the http now, or is it the 8 megs of just straight csv that's still slow?

With this change, all the stuff should be dropped from pysal to libpysal.

File: spvcm/utils.py [hunk 1]
```diff
@@ -34,6 +34,13 @@ def thru_op(*args, **kws):
     else:
         return args, kws
 
+def zero_op(*args, **kws):
+    """
+    this s a zero op. It takes any arguments, keyword
```
Comment (jGaboardi) [inline spvcm/utils.py]: typo --> "this is a zero op..."

**Comment by jGaboardi** [comment]
Shall I merge this PR or should we tag @sjsrey as a reviewer?

**Comment by ljwolf** [comment]
if it's green, please merge & I'll cut a release.

**Pull request merged**